aiohttp/web_urldispatcher.py

```diff
@@ -163,6 +163,9 @@ def __init__(self, name, prefix, directory, *,
             raise ValueError(
                 "No directory exists at '{}'".format(self._directory))
 
+        if os.environ.get("AIOHTTP_NOSENDFILE") == "1":
```

Comment:
It was fixed after the merge: https://github.com/KeepSafe/aiohttp/commit/9dc529b3faabec80698d6de138c40c4ec90c1dee